AudioKit/Core Classes/AKManager.m

```diff
@@ -116,15 +112,20 @@ - (instancetype)init {
         _isBatching = NO;
         
         _orchestra = [[AKOrchestra alloc] init];
+        
+        NSString *inputOption = [NSString stringWithFormat:@"-i %@", AKSettings.shared.audioInput];
+#ifdef TRAVIS_CI
+        inputOption = @"";
+#endif
 
         _options = [NSString stringWithFormat:
                     @"-o %@           ; Write sound to the host audio output\n"
                     "--expression-opt ; Enable expression optimizations\n"
                     "-m0              ; Print raw amplitudes\n"
-                    "-i %@            ; Request sound from the host audio input device",
-                    AKSettings.shared.audioOutput, AKSettings.shared.audioInput];
+                    "%@\n",
+                    AKSettings.shared.audioOutput, inputOption];
         
-        _csdFile = [NSString stringWithFormat:@"%@/AudioKit-%@.csd", NSTemporaryDirectory(), @(getpid())];
+        _csdFile = [NSString stringWithFormat:@"%@/AudioKit.csd", NSTemporaryDirectory()];
```

Comment:
AudioKit produces the same file over and over again, it's never different for different builds, so I got sick of their being a lot of files in that directory.  I have no strong opinion, so if you want it back, I can put it back.